linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_shadow/rule.yml

```diff
@@ -45,4 +45,5 @@ template:
         filegid@ubuntu1604: '42'
         filegid@ubuntu1804: '42'
         filegid@ubuntu2004: '42'
+        filegid@sle15: '15'
```

Comment:
Could you please do it in the same way as here:
https://github.com/ComplianceAsCode/content/pull/8554

Reply:
Done